event/src/test/java/com/bazaarvoice/emodb/event/db/astyanax/ChannelAllocationStateTest.java

```diff
@@ -0,0 +1,143 @@
+package com.bazaarvoice.emodb.event.db.astyanax;
+
+import com.google.common.collect.Iterators;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertTrue;
+
+public class ChannelAllocationStateTest {
+
+    private static final Logger _log = LoggerFactory.getLogger(ChannelAllocationStateTest.class);
+
+    @Test
+    public void allocateOversizeEvent() {
+        List<Integer> sizes = new ArrayList<Integer>();
+        for (int i=0; i<100; i++) {
+            sizes.add(64);
+        }
+        sizes.add(Constants.MAX_EVENT_SIZE_IN_BYTES + 1);
+        ChannelAllocationState channelAllocationState = new ChannelAllocationState();
+        try {
+            channelAllocationState.attachAndAllocate(mock(SlabRef.class), Iterators.peekingIterator(sizes.iterator()));
+            assertTrue(false, "ERROR: No exception thrown when oversize event allocated");
+        } catch (IllegalArgumentException e) {
+            _log.info("SUCCESS: IllegalArgumentException thrown when oversize event allocated");
+        } catch (Exception e) {
+            assertTrue(false, "ERROR: " + e.getClass().getName() + " thrown when oversize event allocated");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "ERROR: " 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)